### PR TITLE
Update docs for imports, logging, and events

### PR DESCRIPTION
**Pull request by adiroiban**
## Proposed change

A quick update based on some questions over the forum.

This is just a first PR.
I plan to add a follow up PR that will include an example in which the `hass.states` is used for persistence across multiple script invocations. 

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @adiroiban, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#24179).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by adiroiban** [comment]
Docs at https://deploy-preview-24179--home-assistant-docs.netlify.app/integrations/python_script/

**Comment by homeassistant** [comment]
Hi @adiroiban, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#24179).

Once you do that we will be able to review and accept this pull request.

Thanks!

**Comment by homeassistant** [comment]
Hi @adiroiban, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#24179).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/python_script.markdown [hunk 1]
```diff
@@ -16,6 +16,12 @@ This integration allows you to write Python scripts that are exposed as services
 | `hass` | The Home Assistant object. Access is only allowed to call services, set/remove states and fire events. [API reference][hass-api]
 | `data` | The data passed to the Python Script service call.
 | `logger` | A logger to allow you to log messages: `logger.info()`, `logger.warning()`, `logger.error()`. [API reference][logger-api]
+| `time` | The stdlib `time` available as limited access.
+| `datetime` | The stdlib `datetime` available as limited access.
+| `dt_util` | The ` homeassistant.util.dt` module.
+
+Other imports like `min`, `max` are available as builtins. See the [python_script](https://github.com/home-assistant/core/blob/dev/homeassistant/components/python_script/__init__.py) source code for up to date information on the available objects inside the scritp.
```
Comment (frenck) [inline source/_integrations/python_script.markdown]:
```suggestion
Other imports like `min`, `max` are available as builtins. See the [python_script](https://github.com/home-assistant/core/blob/dev/homeassistant/components/python_script/__init__.py) source code for up-to-date information on the available objects inside the script.
```

File: source/_integrations/python_script.markdown [hunk 2]
```diff
@@ -26,31 +32,80 @@ It is not possible to use Python imports with this integration. If you want to d
 
 </div>
 
-## Writing your first script
+## Writing your first script, reading input and logging the activity
+
+This is a very simple example that does no real work.
+It is created as a first step, to help with:
```
Comment (frenck) [inline source/_integrations/python_script.markdown]:
```suggestion
This is a simplified example that does no real work.
It is created as a first step, to help with:
```

File: source/_integrations/python_script.markdown [hunk 3]
```diff
@@ -26,31 +32,80 @@ It is not possible to use Python imports with this integration. If you want to d
...
+
+- demonstrate how to setup the script
+- how to process the input data
+- how to log the script activity
+- how to troubleshoot / manually call the script.
```
Comment (frenck) [inline source/_integrations/python_script.markdown]:
```suggestion
- Demonstrate how to setup the script
- How to process the input data
- How to log the script activity
- How to troubleshoot / manually call the script.
```

File: source/_integrations/python_script.markdown [hunk 4]
```diff
@@ -26,31 +32,80 @@ It is not possible to use Python imports with this integration. If you want to d
...
-- Start Home Assistant
-- Call your new {% my developer_call_service service="python_script.hello_world" %} service (with parameters) from the {% my developer_services %}. 
+- Start Home Assistant to reload the script configuration.
+- Call your new {% my developer_call_service service="python_script.hello_world" %} service (with parameters) from the {% my developer_services %}, using the YAML mode. 
 
 ```yaml
-name: you
+service: python_script.hello_world
+data:
+  name: Input-Text
```
Comment (frenck) [inline source/_integrations/python_script.markdown]:
```suggestion
  name: "Input-Text"
```

File: source/_integrations/python_script.markdown [hunk 5]
```diff
@@ -100,6 +155,12 @@ Available services: `reload`.
...
 ### Service `python_script.reload`
 
-Reload all available python_scripts from the `<config>/python_scripts` folder. Use this when creating a new Python script and you're not restarting Home Assistant.
+Reload all available python_scripts from the `<config>/python_scripts` folder.
+
+Use this when creating a new Python script and you're not restarting Home Assistant.
+
+Use this after updating the `<config>/python_scripts/services.yaml` file. 
+
+You don't have to call this service when you edit an existing Python script.
```
Comment (frenck) [inline source/_integrations/python_script.markdown]:
```suggestion
Reload all available python_scripts from the `<config>/python_scripts` folder, as a quicker alternative to restarting Home Assistant.

Use this when creating a new Python script, or after updating the `<config>/python_scripts/services.yaml` file. 

You don't have to call this service when you change an existing Python script.
```

File: source/_integrations/python_script.markdown [hunk 6]
```diff
@@ -26,31 +32,80 @@ It is not possible to use Python imports with this integration. If you want to d
...
 </div>
 
+## Triggering events
+
+The following example shows how to trigger a custom event over the `hass.bus`.
+
+We will use the same `hello_world.py` file.
+Edit the file and add this at the end.
+No need to restart Home Assistant or reload any configuration.
+The script is (re)loaded at each call.
```
Comment (frenck) [inline source/_integrations/python_script.markdown]:
```suggestion
This example uses the `hello_world.py` from the previous example.
Edit the file and the code listed below to the end of the file.
There is no need to reload the configuration or restart Home Assistant.
```

File: source/_integrations/python_script.markdown [hunk 7]
```diff
@@ -26,31 +32,80 @@ It is not possible to use Python imports with this integration. If you want to d
...
+
+```python
+hass.bus.fire("hello_world_event", {"wow": "from a Python script!"})
+```
+
+Running this script show absolutely no output on the screen.
+From a separate tab (session) you will need to go to `Developer Tools -> Events` and at `Listen to events` type `hello_world_event` and then press `Start listening`.
+You should see something like this:
```
Comment (frenck) [inline source/_integrations/python_script.markdown]:
```suggestion
This script doesn't output anything. However, you can view the events being fired in the Developer tools.

From a separate browser window or tab, go to `Developer Tools -> Events` and at `Listen to events` type `hello_world_event` and then press `Start listening`. You should see something like this:
```

**Review by frenck — APPROVED** [review]
Thanks, @adiroiban 👍

**Pull request merged**